plugins/woocommerce/client/admin/client/wp-admin-scripts/email-editor-integration/templates/template_sender_panel.tsx

```diff
@@ -0,0 +1,49 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Panel, PanelBody, PanelRow, TextControl } from '@wordpress/components';
+
+function TemplateSenderPanel() {
+	return (
+		<Panel className="woocommerce-email-sidebar-template-settings-sender-options">
+			<PanelBody>
+				<PanelRow>
+					<div>
+						<h2>{ __( 'Sender Options', 'woocommerce' ) }</h2>
+						<p>
+							{ __(
+								'This is how your sender name and email address would appear in outgoing WooCommerce emails.',
+								'woocommerce'
+							) }
+						</p>
+					</div>
+				</PanelRow>
+				<PanelRow>
+					<TextControl
+						className="woocommerce-email-sidebar-template-settings-sender-options-input"
+						label={ __( '“from” name', 'woocommerce' ) }
```

Comment:
This translation would perhaps use a comment for translators. Maybe something like:
```
/* translators: Label for the sender's name in email settings. */
```
I would add it also for “from” email